anitya/lib/utilities.py

```diff
@@ -0,0 +1,508 @@
+# -*- coding: utf-8 -*-
```

Comment:
The project doesn't seem to have an overall copyright claim. I recommend claiming copyright on this file (and I recommend correcting this for the rest of the project too). Don't forget that this code came from existing files, so the copyright claim should take the history of those files into account for ownership and dates.